Fixed typos in shading_language.rst

kjav opened the pull request:
Fixes typos present in 3.0 docs.

vnen commented on tutorials/shading/shading_language.rst:
```diff
@@ -807,7 +807,7 @@ Fragment Built-Ins
 | in vec3 **VERTEX**               | Vertex that comes from vertex function, in view space.                                           |
 +----------------------------------+--------------------------------------------------------------------------------------------------+
-| in vec4 **FRAGCOORD**            | Fragment cordinate, pixel adjusted.                                                              |
+| in vec4 **FRAGCOORD**            | Fragment coordinate, pixel adjusted.                                                              |
```
Is the table still aligned after this change?

kjav commented:
@vnen I think I have fixed the issue you raised - I deleted a space character so that the line is now the same length as the previous entry, including the trailing whitespace.

mhilbrunner commented:
Thanks for your contribution!
